client/view/Alert/Alert.js

```diff
@@ -12,9 +12,14 @@ export function buildError(root, message) {
   return error
 }
 
-export function buildWarning(root, message) {
+export function buildWarning(root, message, fixed) {
   let warning = createTag('div', ['Alert', 'is-warning'])
+  let fix = createTag('a', ['QueryLink', 'is-fix'])
```

Comment:
We need also to move button to the right side if the warning.

I suggest wrapping button to `div.Alert_action`

Reply:
if I correctly understood, need to move button like on screenshot?
<img width="1428" alt="Screenshot 2022-09-29 at 13 21 12" src="https://user-images.githubusercontent.com/22856389/192998422-9171d7d9-ff91-40ee-ac61-d9b661d50597.png">

Reply:
1. No, we will have a lot of unused space in the top-right corner and bottom-left. Put the button to the top-right corner
2. Don't forget to check contrast in light and dark mode
3. In our design language all buttons have background. I don't see it on this screenshot.